xfce-test

```diff
@@ -486,7 +498,7 @@ elif [ $mode == "call" ]; then
     RESUME_MODE="--attach"
 fi
 
-DOCKER_MODES="${DOCKER_MODES} --volume /etc/timezone:/etc/timezone"
+#DOCKER_MODES="${DOCKER_MODES} --volume /etc/timezone:/etc/timezone"
```

Comment:
This file doesn't exist on Arch host, and therefore is not even needed inside an Archlinux docker environment.

We can probably add an if statement here to verify if it exists before attempting to add the volume.

Reply:
Fixed with a check